docker-compose.yaml

```diff
@@ -3,6 +3,7 @@ version: '3'
 services:
   postgres-data:
```

Comment:
we should align service name with consumer name for te two databases

Reply:
changed them